manifests/init.pp

```diff
@@ -198,13 +340,45 @@
   Optional[String[1]] $extract_command,
   Boolean $manage_config,
   Stdlib::Absolutepath $usershell,
-  Hash $extra_alerts    = {},
-  Hash $config_hash     = {},
-  Hash $config_defaults = {},
-  String $os            = downcase($facts['kernel']),
+  Optional[String] $web_listen_address                                          = undef,
+  Optional[String] $web_read_timeout                                            = undef,
+  Optional[String] $web_max_connections                                         = undef,
+  Optional[String] $web_route_prefix                                            = undef,
+  Optional[String] $web_user_assets                                             = undef,
+  Optional[Boolean] $web_enable_lifecycle                                       = undef,
```

Comment:
for those boolean parameters, can you remove the `Optional[]` and set the default to true/false?

Reply:
can do!